src/http/one/TeChunkedParser.cc

```diff
@@ -125,6 +125,10 @@ Http::One::TeChunkedParser::parseChunkMetadataSuffix(Tokenizer &tok)
     // Code becomes much simpler when incremental parsing functions throw on
     // bad or insufficient input, like in the code below. TODO: Expand up.
     try {
+        // Bug 4492: IBM_HTTP_Server sends SP after chunk-size.
+        // No ParseBws() here because it may consume CR required further below.
+        ParseStrictBws(tok);
```

Comment:
This being a very specific and CVE related case, I see much benefit in avoiding the new special `ParseBws()` logic.

Please instead prefer paranoia level of clarity:
```suggestion
        // Also, this is not BWS. It is explicit "MUST reject" violation of RFC 9112 HTTP/1.1.
        (void)tok.skipOne(CharacterSet::WSP);
```

Reply:
Hello Amos. As Alex mentioned in a0d4fe17 , there has been seen two spaces after chunk-size. Suggested change  therfor would not fix such case.

Reply:
In addition to what Luboš has [said](https://github.com/squid-cache/squid/pull/1914/files#r1796842949), skipOne() and skipAll() do not work well for incremental parsing of bad/prohibited-by-protocol input, as was already pointed out in the [initial review](https://github.com/squid-cache/squid/pull/1914#discussion_r1795888509) of this code. Please do not go back to the already rejected code.

Reply:
Very well, skip twice then.

Reply:
> Very well, skip twice then.

Skipping twice would not address my [concern](https://github.com/squid-cache/squid/pull/1914#discussion_r1796949281). Moreover, correctly skipping at most twice requires heroic efforts including creation of new parsing state. Simply calling skipOnce() twice would not accomplish that.